core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterService.java

```diff
@@ -414,7 +383,6 @@ public Cluster prepareCluster(Collection<HostGroup> hostGroups, Blueprint bluepr
         cluster.getHostGroups().clear();
         cluster.getHostGroups().addAll(hostGroups);
         LOGGER.debug("Cluster requested [BlueprintId: {}]", cluster.getBlueprint().getId());
-        cluster.setStatus(REQUESTED);
         cluster.setStack(stack);
```

Comment:
shouldn't we set the status to the same as stack status?

Reply:
Its just the initialization of the cluster entity, every other status change is handled by _StackUpdater_ class.